android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -44,6 +46,50 @@ class JsonConverter {
                     playerConfig.playbackConfig.isTimeShiftEnabled = playbackConfigJson.getBoolean("isTimeShiftEnabled")
                 }
             }
+            if (json.hasKey("tweaksConfig")) {
+                val tweaksConfigJson = json.getMap("tweaksConfig")
+                if (tweaksConfigJson?.hasKey("timeChangedInterval") == true) {
+                    playerConfig.tweaksConfig.timeChangedInterval = tweaksConfigJson.getDouble("timeChangedInterval")
+                }
+                if (tweaksConfigJson?.hasKey("bandwidthEstimateWeightLimit") == true) {
+                    playerConfig.tweaksConfig.bandwidthEstimateWeightLimit = tweaksConfigJson.getInt("bandwidthEstimateWeightLimit")
+                }
+                if (tweaksConfigJson?.hasKey("devicesThatRequireSurfaceWorkaround") == true) {
+                    val devices = tweaksConfigJson.getMap("devicesThatRequireSurfaceWorkaround")
+                    val deviceNames = devices?.getArray("deviceNames")
+                    val modelNames = devices?.getArray("modelNames")
+                    val descriptions = ArrayList<DeviceDescription>()
+                    if (deviceNames != null) {
+                        for (i in 0 until deviceNames.size()) {
+                            descriptions.add(DeviceName(deviceNames.getString(i)))
+                        }
+                    }
+                    if (modelNames != null) {
+                        for (i in 0 until modelNames.size()) {
+                            descriptions.add(ModelName(modelNames.getString(i)))
+                        }
+                    }
+                    playerConfig.tweaksConfig.devicesThatRequireSurfaceWorkaround = descriptions
```

Comment:
This does not look very kotlinish, but I also noticed that these collections from RN do not follow any "native" interface like `Collection` or `Iterable`...
Do you know if there is some project that extents RN with kotlin friendly extensions? 🤔 

It could look something like this
```suggestion
                    val deviceNames = devices?.getArray("deviceNames")
                        ?.toStringList()
                        ?.map { DeviceName(it) }
                        ?: emptyList()
                    val modelNames = devices?.getArray("modelNames")
                        ?.toStringList()
                        ?.map { ModelName(it) }
                        ?: emptyList()
                    playerConfig.tweaksConfig.devicesThatRequireSurfaceWorkaround = deviceNames + modelNames
```
```kotlin
private fun ReadableArray.toStringList() =  (0 until size()).map {
    getString(it)
}
```

Reply:
I created a `toList` extension on `ReadableArray` following the code you shared. Just tried to generalize it a bit to support other types beyond `String` 🙂

3353908c7551bb03a0074c31e2aad60bc7ddee4e